packages/now-cli/src/commands/secrets.js

```diff
@@ -286,13 +294,11 @@ function readConfirmation(secret) {
       hsep: ' '.repeat(6),
     });
 
-    process.stdout.write(
-      '> The following secret will be removed permanently\n'
-    );
+    process.stdout.write('The following secret will be removed permanently\n');
```

Comment:
```suggestion
    process.stdout.write(`The following secret will be removed permanently from ${chalk.bold(
        contextName
      )}\n`);
```

Also, is there any reason we're not using `output.log()` for this?